README.md

```diff
@@ -382,11 +382,11 @@ for await (const [key, value] of db.iterator()) {
 
 > :pushpin: The key structure is equal to that of [`subleveldown`](https://github.com/Level/subleveldown) which offered sublevels before they were built-in to `abstract-level`. This means that an `abstract-level` sublevel can read sublevels previously created with (and populated by) `subleveldown`.
 
-Internally, sublevels operate on keys that are either a string, Buffer or Uint8Array, depending on parent database and choice of encoding. Which is to say: binary keys are fully supported. The `name` must however always be a string and can only contain ASCII characters.
+Internally, sublevels operate on keys that are either a string, Buffer or Uint8Array, depending on parent database and choice of encoding. Which is to say: binary keys are fully supported. The `name` must however always be a string.
 
 The optional `options` object may contain:
 
-- `separator` (string, default: `'!'`): Character for separating sublevel names from user keys and each other. Must sort before characters used in `name`. An error will be thrown if that's not the case.
```

Comment:
> Must sort before characters used in `name`

Hm this is still true, because otherwise nested sublevels don't sort correctly. I should add a test for that scenario, and then restore the `LEVEL_INVALID_PREFIX` check (but more relaxed, only for the low bytes).